spec/teacher_spec.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../teacher'
+
```

Comment:
- [ ] Please look through your **_classes_** again and ensure you implement tests for all the methods in them.

Reply:
All require test for all methods are done